src/style.css

```
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+/* .container {
+  margin: 32px 22px;
+  width: 80px;
+  height: 78px;
+} */
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  width: 80px;
+  height: 12px;
+}
```

```suggestion
.stars {
  display: flex;
}
```
do not set widht and height for stars. 
content of stars will make size of row
margin and padding will be redundant after change ul to div